libs/utils/env.py

```diff
@@ -410,50 +410,56 @@ def _init_target(self, force = False):
         if 'board' not in self.conf:
             self.conf['board'] = 'UNKNOWN'
 
+        board_name = self.conf['board'].upper()
+
         # Initialize TC2 board
-        if self.conf['board'].upper() == 'TC2':
+        if board_name == 'TC2':
             platform = devlib.platform.arm.TC2()
             self.__modules = ['bl', 'hwmon', 'cpufreq']
 
         # Initialize JUNO board
-        elif self.conf['board'].upper() in ('JUNO', 'JUNO2'):
+        elif board_name == 'JUNO':
```

Comment:
Cannot we still try to keep one section per "board":
```python
 elif board_name in in ('JUNO', 'JUNO2'):
    # ...
    if board_name == JUNO:
        self.nrg_model = juno_r0_energy
    elif board_name == JUNO2:
        self.nrg_model = juno_energy
    # ...
```

Reply:
Done.